packages/openactive-integration-tests/test/features/restrictions/booking-window/implemented/opportunity-outside-range-c1-c2-test.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const chakram = require('chakram');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2 } = require('../../../../shared-behaviours');
+
+/**
+ * @typedef {import('chakram').ChakramResponse} ChakramResponse
+ */
+
+/**
+ * @param {C1|C2} stage
+ * @param {() => ChakramResponse} responseAccessor This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturn409Conflict(stage, responseAccessor) {
+  it('should return 409', () => {
+    stage.expectResponseReceived();
+    chakram.expect(responseAccessor()).to.have.status(409);
+  });
+}
+
+/**
+ * @param {C1|C2} stage
+ * @param {() => ChakramResponse} responseAccessor This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturnOpportunityOfferPairNotBookableError(stage, responseAccessor) {
+  it('should return OpportunityOfferPairNotBookableError', () => {
```

Comment:
Note this should only be checking for errors in non-control items, and no errors in control items. A mirror of this example:

https://github.com/openactive/openactive-test-suite/blob/07f80ffa48d23e2d6916827d3815e6ddeed37973/packages/openactive-integration-tests/test/features/core/availability-check/implemented/opportunity-full-test.js#L48-L77